spec/features/atom/issues_spec.rb

```diff
@@ -24,10 +24,10 @@
       it "should render atom feed" do
         visit project_issues_path(project, :atom, private_token: user.private_token)
 
-        response_headers['Content-Type'].should have_content("application/atom+xml")
-        body.should have_selector("title", text: "#{project.name} issues")
-        body.should have_selector("author email", text: issue.author_email)
-        body.should have_selector("entry summary", text: issue.title)
+        expect(response_headers['Content-Type']).to have_content("application/atom+xml")
+        expect(body).to have_selector("title", text: "#{project.name} issues")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.